recipes/squashfuse/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "squashfuse" %}
+{% set version = "0.1.103" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/vasi/squashfuse/releases/download/{{ version }}/squashfuse-{{ version }}.tar.gz
+  sha256: 42d4dfd17ed186745117cfd427023eb81effff3832bab09067823492b6b982e7
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - pkg-config
+    - make
+  host:
+    - libfuse 2.*
```

Comment:
Should we add this to the global pinning file?

Reply:
No, this needs an old major release and isn't working with the current major release.